frontend/src/components/SprintsPage/SprintsDashboard.css

```diff
@@ -0,0 +1,38 @@
+.sprints-container {
+  margin: 20px;
+}
+
+.add-sprint-container {
+  margin-bottom: 40px;
+}
+
+.sprint-list-container {
+  margin-top: 20px;
+  border-top: 2px solid #ccc;
+  padding-top: 20px;
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse; 
+}
+
+th, td {
+  padding: 8px; 
+  text-align: left; 
+  border-bottom: 1px solid #ddd; 
+}
+
+th {
+  background-color: #f2f2f2; 
+}
+
+tr:hover {
+  background-color: #f5f5f5; 
+}
+
+.spacer {
+  height: 45px; 
+}
+
+  
```

Comment:
Add newline at end of file (in this case remove redundant whitespace)

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b